shared/actions/chat.js

```diff
@@ -225,8 +230,8 @@ function openTlfInChat (tlf: string): OpenTlfInChat {
   return {type: 'chat:openTlfInChat', payload: tlf}
 }
 
-function startConversation (users: Array<string>): StartConversation {
-  return {type: 'chat:startConversation', payload: {users}}
+function startConversation (users: Array<string>, forceImmediate: ?boolean = false): StartConversation {
+  return {type: 'chat:startConversation', payload: {forceImmediate: !!forceImmediate, users}}
```

Comment:
You can just do:

`forceImmediate?: boolean = false)...` and then you don't need to do that!